dvc/tree/base.py

```diff
@@ -90,6 +90,10 @@ def hash_jobs(self):
             or self.HASH_JOBS
         )
 
+    @cached_property
+    def enable_dos2unix(self):
+        return self.repo and self.repo.config["core"].get("dos2unix")
```

Comment:
I don't think this really belongs in `BaseTree` long term, but currently we compute MD5's from both `RepoTree` and `LocalTree`